Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_CMDB_Enrichment.yml

```diff
@@ -0,0 +1,537 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: c5291e8b-29d3-42de-839b-a4a0d1e08570
+    type: title
+    task:
+      id: c5291e8b-29d3-42de-839b-a4a0d1e08570
+      version: -1
+      name: Add CMDB Record
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -880,
+          "y": 1050
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: fe030616-653d-4728-8b9a-1946edb756b2
+    type: title
+    task:
+      id: fe030616-653d-4728-8b9a-1946edb756b2
+      version: -1
+      name: Fetch CMDB Record
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -410,
+          "y": 1050
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 49d88a89-9b8a-4c2f-8768-08d6b584d039
+    type: regular
+    task:
+      id: 49d88a89-9b8a-4c2f-8768-08d6b584d039
+      version: -1
+      name: Fetch Related CMDB CI by ID
+      description: Query attributes and relationship information for a specific record.
+      script: ServiceNow CMDB|||servicenow-cmdb-record-get-by-id
+      type: regular
+      iscommand: true
+      brand: ServiceNow CMDB
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      class:
+        simple: cmdb_ci
+      sys_id:
+        simple: ${ServiceNowCMDB.Record.Attributes.cmdb_ci.value}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -410,
+          "y": 1630
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: fb51a5ef-1c13-43eb-83dd-5b9b32e32067
+    type: regular
+    task:
+      id: fb51a5ef-1c13-43eb-83dd-5b9b32e32067
+      version: -1
+      name: Save ServiceNow CMDB CI to Notes
+      description: Convert an array to a nice table display. Usually, from the context.
+      scriptName: ToTable
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      columns:
+        simple: name,sys_class_name,sys_id,asset.display_value,asset.link,asset.value
+      data:
+        simple: |2-
+
+          ${ServiceNowCMDB.Record.Attributes}
```

Comment:
@andrew-paloalto Please change to a filtered value structure (and remove the unneeded new line prefix).